### PR TITLE
Add Exponential Distribution and others 

### PR DESCRIPTION
**Pull request by basil08**
Implemented Exponentialdistribution.py 
Also, added a .gitignore file 

And corrected some basic factual error in documentation.

### CODE REVIEW

**Review by hot9cups — APPROVED** [review]
This looks great. Will merge soon.

File: test.py [hunk 1]
```diff
@@ -82,6 +83,34 @@ def test_add(self):
...
+        self.assertEqual(self.exponential.calculate_mean(), \
+                (1.0/0.25) , 'calculated mean not as expected')
+
+    def test_stdevcalculation(self):
+        self.assertEqual(self.exponential.calculate_stdev(), \
+                (1.0/0.25), 'calculated standard deviation incorrect')
+
+    def test_pdf(self):
+        self.assertEqual(round(self.exponential.pdf(1), 5, 0.19470, \
+                'pdf function does not give expexted result')
```
Comment (hot9cups) [inline test.py:109]: Minor type here.
'pdf function does not give **expected** result'.

**Comment by hot9cups** [comment]
Everything looks great. Could you confirm that you've run the tests and everything works fine?

**Comment by basil08** [comment]
Yeah it does.

**Review by hot9cups — APPROVED** [review]

**Review by hot9cups — APPROVED** [review]
The PR accomplished the following:
1)Implementation of Exponential Distribution
2)Addition of .gitignore file
3)Documentation fixes
4)Addition of unittests for Exponential Distribution

That was a lot of work! Kudos and thanks a lot for taking the time to contribute! 
Are you okay with being mentioned as a major contributor when I release the next version of the library(which will happen in a couple of days)? :)

**Comment by basil08** [comment]
Oh that would be great!
Thank you so much!

By the way, Kudos to you for the meticulously documented code. It helped me
so much get an understanding of the code as I am a beginner.

**Comment by hot9cups** [comment]
Great! I'll notify you when the next release is out 👍 

Ah, thank you. Documentation was a primary focus since I've struggled a lot in the past with libraries owing to their abysmal documentary, I didn't my library to follow suit. 
I believe anyone can write good documentation if they put in enough effort into it.

Infact, would you like to go ahead and add in usage of Exp distribution that you just implemented, to usage.md, to get a feel for it? :)

**Comment by basil08** [comment]
Sure. I'll open a PR tomorrow!

**Pull request merged**